site/_docs/file_adapter.md

```diff
@@ -158,7 +158,7 @@ sqlline> select * from sales.emps;
 ## Mapping tables
 
 Now for a more complex example. This time we connect to Wikipedia via
-HTTP, read pages for US states and cities, and extract data from HTML
+HTTP, read pages for US states and cities and extract data from HTML
```

Comment:
The original (with comma) is correct.

See https://en.wikipedia.org/wiki/Eats,_Shoots_%26_Leaves.